atomate/vasp/firetasks/run_calc.py

```diff
@@ -193,10 +245,12 @@ def run_task(self, fw_spec):
         doping = self.get("doping", None)
         soc = self.get("soc", False)
 
-        vasprun, outcar = get_vasprun_outcar(".", parse_dos=True, parse_eigen=True)
+        vasprun, outcar = get_vasprun_outcar(
```

Comment:
I think these sort of changes where you are splitting one line into 2 just to satisfy a line length limit does not make sense. The idea behind an 80 character limit is to help make code more readable and splitting these one line function calls to two lines undermines that, in my opinion.